### ISSUE TITLE
Not compressed

### ISSUE THREAD

**Issue by Bashima** [DwangoMediaVillage/keras_compressor#13]
There is no compression. I have a the following layers conv1D>batch_norm>activation> max_pool

**Command: keras-compressor.py --error 0.1 with.h5 com.h5**

**OUTPUT**
Using TensorFlow backend.
2018-10-16 13:09:04.704740: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.convolutional.Conv1D object at 0x7fa81f200080>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.normalization.BatchNormalization object at 0x7fa81f1eab00>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.core.Activation object at 0x7fa81f1eab38>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.pooling.MaxPooling1D object at 0x7fa81f1ae710>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.convolutional.Conv1D object at 0x7fa81f1ea898>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.normalization.BatchNormalization object at 0x7fa81f0e9080>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.core.Activation object at 0x7fa81f1a2e48>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.pooling.MaxPooling1D object at 0x7fa81f11e048>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.convolutional.Conv1D object at 0x7fa81f0e9ef0>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.normalization.BatchNormalization object at 0x7fa81f0381d0>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.core.Activation object at 0x7fa81f085ef0>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.pooling.MaxPooling1D object at 0x7fa81edcb358>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.convolutional.Conv1D object at 0x7fa81edddbe0>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.normalization.BatchNormalization object at 0x7fa81ed5ee10>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.core.Activation object at 0x7fa81ed2a710>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.pooling.MaxPooling1D object at 0x7fa81f038b00>
INFO:keras_compressor.compressor:factorizer not found layer:<keras.layers.core.Lambda object at 0x7fa81ed2ae48>
INFO:keras_compressor.compressor:factorizer found layer:<keras.layers.core.Dense object at 0x7fa81ed1e860> factorizer:<class 'keras_compressor.factorizers.svd.SVDFactorizer'>
INFO:keras_compressor.compressor:factorization failed layer:<keras.layers.core.Dense object at 0x7fa81ed1e860> factorizer:<class 'keras_compressor.factorizers.svd.SVDFactorizer'>
Compressed model
    before #params              561,162
    after  #params              561,162 (0.00%)

**Comment by suzy8808** [comment]
My model isn't compressed too

**Comment by EricHuiK** [comment]
My model isn't compressed too （keras-retinaNet）
